modules/swagger-parser/src/main/java/io/swagger/parser/util/SwaggerDeserializer.java

```diff
@@ -819,6 +820,10 @@ public Model definition(ObjectNode node, String location, ParseResult result) {
             JsonNode ap = node.get("additionalProperties");
             if (ap != null && ap.getNodeType().equals(JsonNodeType.OBJECT)) {
                 impl.setAdditionalProperties(Json.mapper().convertValue(ap, Property.class));
+            } else if ("object".equalsIgnoreCase(type) && (ap == null || "true".equalsIgnoreCase((String) ap.asText()))) {
```

Comment:
Should we set additionalProperties true for composed schemas also?
What does the master branch of swagger parser do?